product_supplierinfo_for_customer/__manifest__.py

```diff
@@ -17,11 +17,11 @@
         "purchase",
```

Comment:
Why this one depends on purchase? I don't think the module must depend on it, and more wanting to use it for customers